tools/cdb2sql/cdb2sql.c

```diff
@@ -887,6 +891,26 @@ static void replace_args(int argc, char *argv[])
     }
 }
 
+
+static int int_handler_cnt = 0;
+
+/* If ctrl_c was pressed to clear existing line and go to new line
```

Comment:
Is that something that you see in any other readline type prompt? Bash certainly does not do anything on ctrl-d (when line has characters). Maybe PG or mysql do it?